app/scripts/views/mixins/password-strength-mixin.js

```diff
@@ -34,14 +32,7 @@ export default function (config = {}) {
     ],
 
     afterRender () {
-      const experimentGroup = this._getPasswordStrengthExperimentGroup();
-      if (experimentGroup) {
-        this.createExperiment(EXPERIMENT_NAME, experimentGroup);
-
-        if (experimentGroup === DESIGN_F_GROUP) {
-          return this._setupDesignF();
-        }
-      }
+      this._setupDesignF();
     },
 
     _setupDesignF () {
```

Comment:
Should we just move the body of this function into `afterRender` or does `_setupDesignF` still have some tangible meaning to us post-experiment?